ui/src/app/cohort-search/tree/tree.component.tsx

```diff
@@ -188,6 +195,7 @@ export const CriteriaTree = withCurrentWorkspace()(class extends React.Component
           <ClrIcon style={{color: colors.white}} className='is-solid' shape='exclamation-triangle' />
           Sorry, the request cannot be completed. Please try again or contact Support in the left hand navigation.
         </div>}
+        <div style={!this.showHeader ? styles.node : {...styles.node, borderTop: 'none'}}>
```

Comment:
I think we need to have no border here when `showHeader` is false (Physical Measurements, Surveys, Visits) like we currently do in test.

Reply:
Done and have updated the screen shot in description